packages/extension/src/store/synchnorousCookieStore.ts

```diff
@@ -49,7 +100,8 @@ class SynchnorousCookieStore {
       devToolsOpenState: boolean;
       popupOpenState: boolean;
       newUpdates: number;
-      portRef: any;
+      frameIDURLSet: Record<string, string[]>;
+      parentChildFrameAssociation: Record<string, string>;
```

Comment:
Explaining these keys in JSdoc will be helpful, as it takes some time to find callbacks and use cases.

Reply:
Fixed in latest commits.